api/examples/python/README.md

```diff
@@ -8,16 +8,13 @@ notifications.
 First, install the dependencies.
 
 ```
-sudo /usr/bin/python -m pip install websocket-client
-sudo /usr/bin/python -m pip install protobuf
-
+mkvirtualenv iterm2api
```

Comment:
OK to add a section describing how to do it using virtualenv as an alternative to `sudo python` but I don't want to insist on more deps than are necessary. Dependencies are the root of all frustration.